api/turing/api/ensembling_job_api_it_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration
```

Comment:
Same for this, why is this changed?

Reply:
^Same as above, some remnants left by the IDE probably.